src/main/scala/diplomaticobjectmodel/DiplomaticObjectModelUtils.scala

```diff
@@ -166,15 +166,17 @@ object DiplomaticObjectModelAddressing {
     width: Int,
     depth: BigInt,
     granWidth: Int,
-    uid: Int
+    uid: Int,
+    rtlModule: OMRTLModule = OMRTLModule("")
```

Comment:
Since you have set a default value for all the fields on the `OMRTLModule` case class, why not just call `OMRTLModule()` with no arguments?

Reply:
Just an oversight from before the defaults. Thx for catching.